### PR TITLE
Virtualized Scroller for icon interface

### PR DESCRIPTION
**Pull request by jacobrienstra**
It works on search, but it's a bit slow. Not sure if that's storybook or the code. I imagine with all the observing happening this might just be the tradeoff...but I'm not sure

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "cropperjs": "^1.5.7",
     "date-fns": "^2.14.0",
     "diff": "^4.0.2",
+    "intersection-observer": "^0.10.0",
```
Comment (rijkvanzanten) [inline package.json:34]: What's this new dependency? `IntersectionObserver` is supported in all our deployment targets: https://caniuse.com/#feat=intersectionobserver

File: src/interfaces/icon/icon.vue [hunk 2]
```diff
@@ -28,21 +28,35 @@
...
-				/>
-			</template>
+			<DynamicScroller
+				:items="filteredIcons"
+				:min-item-size="24"
+				class="scroller"
+				key-field="name"
+			>
+				<template v-slot="{ item, index, active }">
+					<DynamicScrollerItem
```
Comment (rijkvanzanten) [inline src/interfaces/icon/icon.vue:38]: Ahh interesting, I thought the individual icons had to be the scroller item in order to be really efficient in scrolling 🤔

File: src/interfaces/icon/icon.vue [hunk 3]
```diff
@@ -51,6 +65,7 @@
 import icons from './icons.json';
 import { defineComponent, ref, computed } from '@vue/composition-api';
 import formatTitle from '@directus/format-title';
+import 'vue-virtual-scroller/dist/vue-virtual-scroller.css';
```
Comment (rijkvanzanten) [inline src/interfaces/icon/icon.vue:68]: Can we put this in the <style>?

File: src/interfaces/icon/icon.vue (hunk 2 above)
Comment (jacobrienstra) [reply src/interfaces/icon/icon.vue:38]: I thought so too, i can try it again but when I did it with the other plugin the grid styling didn't work

File: src/interfaces/icon/icon.vue (hunk 3 above)
Comment (jacobrienstra) [reply src/interfaces/icon/icon.vue:68]: LOL DUH MY B

File: src/interfaces/icon/icon.vue (hunk 2 above)
Comment (rijkvanzanten) [reply src/interfaces/icon/icon.vue:38]: Yeah I ran into similar issues on my attempt..

File: package.json (hunk 1 above)
Comment (jacobrienstra) [reply package.json:34]: It wasn't working so I added this, I'll try again now that it is working without.

**Pull request closed without merging**